readme.md

```diff
@@ -26,6 +26,7 @@ Elements in beautiful READMEs include, but are not limited to: images, screensho
 - [jakubroztocil/httpie](https://github.com/jakubroztocil/httpie#readme) - Description of what the project does. Demo screenshots. Project logo. TOC for easy navigation. Build badges. Quick and simple installation and usage sections. Includes an examples section.
 - [karan/joe](https://github.com/karan/joe#readme) - Project logo. Clear description of what the project does. GIF demo. Easy install and usage sections.
 - [Martinsos/edlib](https://github.com/Martinsos/edlib#readme) - Informative badges (build, version, publication). Concise description. Feature list. TOC. Screenshots. Concise instructions with examples for building and including in your project. Common code examples to get you started quickly.
+- [Bananut-Electronics/MiDispositivoMIDI_V3](https://github.com/Bananut-Electronics/MiDispositivoMIDI_V3#readme) - Badges. Emoticons. GIF. Index. Pictures for each section. Links for navigation. Licence. Social networks links.
```

Comment:
This needs to be inserted alphabetically based on the organisation name of `Banaut-Electronics` rather than the repo name of `MiDispositivoMIDI_V3` in order to be consistent with the rest of the list.